src/main/java/com/marek/onlinebookstore/dto/order/OrderDto.java

```diff
@@ -0,0 +1,21 @@
+package com.marek.onlinebookstore.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public record OrderDto(
+        @Positive
+        Long id,
+        @Positive
+        Long userId,
+        Set<OrderItemDto> orderItems,
```

Comment:
Consider adding '@NotEmpty' constraint to 'orderItems' to ensure that an order must contain at least one item.